src/linear_algebra/tensor_algebra/grading.lean

```diff
@@ -37,12 +37,13 @@ variables {R M}
 instance graded_algebra :
   graded_algebra ((^) (ι R : M →ₗ[R] tensor_algebra R M).range : ℕ → submodule R _) :=
 graded_algebra.of_alg_hom _
-  (lift _ $ graded_algebra.ι R M)
+  -- while not necessary, the `by apply` makes this elaborate faster
+  (lift R $ by apply graded_algebra.ι R M)
```

Comment:
Better or worse as
```suggestion
  (by apply lift R $ graded_algebra.ι R M)
```
or
```suggestion
  (lift R $ graded_algebra.ι R M : _)
```

Reply:
In fact, after merging latest master, the plain `(lift R $ graded_algebra.ι R M)` seems the fastest!